tests/vasp/sets/test_matpes.py

```diff
@@ -42,9 +42,7 @@ def test_matpes_sets(set_generator: VaspInputGenerator) -> None:
     assert matpes_set.inherit_incar is False
     assert matpes_set.auto_ismear is False
     assert matpes_set.auto_kspacing is False
-    assert matpes_set.force_gamma is True
```

Comment:
was this line failing the test? don't we want to keep it?

Reply:
These aren't relevant for MatPES since it uses KSPACING and not KPOINTS (`force_gamma` only affects KPOINTS), and there's a fixed value of KSPACING (`auto_metal_kpoints` is again only used for KPOINTS)

Reply:
I just thought it would maybe be misleading to people in the future if I left that in, since it implies those params are important to check. Happy to add them back in

Reply:
thanks for explaining! didn't realize `force_gamma` doesn't apply to `kspacing`